packages/safe-ds-lang/src/language/generation/safe-ds-python-generator.ts

```diff
@@ -1083,6 +1090,10 @@ export class SafeDsPythonGenerator {
         throw new Error('Callable of provided call does not exist or is not a declaration.');
     }
 
+    private getClassQualifiedNameForMember(expression: SdsClassMember): string {
+        return `${this.getPythonNameOrDefault(AstUtils.getContainerOfType(expression.$container, isSdsDeclaration)!)}.${this.getPythonNameOrDefault(expression)}`;
```

Comment:
Classes can be nested, which is not handled here:

```
class Outer {
    class Nested {
        @Pure
        static fun f() -> result: Boolean
    }
}
```

Reply:
If understood correctly, this should now be fixed